docs/src/modules/sandbox/StackBlitz.ts

```diff
@@ -64,8 +65,9 @@ export default defineConfig({
     'index.html': CRA.getHtml({ ...demoData, main: `/src/index.${ext}` }),
     'package.json': JSON.stringify(
       {
-        name: 'mui-demo',
+        version: process.env.LIB_VERSION,
         private: true,
+        description,
```

Comment:
Because people might eject out of StackBlitz, the description we add to the StackBlitz, seems also relevant in the package.json.